docs/introduction/models.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "3D Models"
+type: introduction
+layout: docs
+parent_section: introduction
+order: 8.75
+---
+
+*work in progress*
+
+## Model File Formats
+
+*work in progress*
+
+## Animating Models
+
+*work in progress*
```

Comment:
What are you looking for here?  Like a Mixamo → A-Frame workflow? Or just how to play animation on a model that already has them baked in?

This video has been recommended to me: https://www.youtube.com/watch?v=Gb152Qncn2s

And then of course: http://unboring.net/workflows/animation.html